src/sagemaker/jumpstart/cache.py

```diff
@@ -211,7 +212,23 @@ def _get_manifest_key_from_model_id_semantic_version(
                 f"{model_version_to_use_incompatible_with_sagemaker} of {model_id}."
             )
             raise KeyError(error_msg)
-        error_msg = f"Unable to find model manifest for {model_id} with version {version}."
+
+        error_msg = f"Unable to find model manifest for {model_id} with version {version}. "
+
+        other_model_id_version = self._select_version(
+            "*", versions_incompatible_with_sagemaker
```

Comment:
can you please add a comment as to why you use this variable.

Reply:
reiterate: please explain why we use a variable called `versions_incompatitble_with_sagemaker` here.